### PR TITLE
Add library `xod-dev/pn532-nfc`

### PR DESCRIPTION
**Pull request by brusherru**
It's an example of wrapping C++ libraries in XOD.
Please, check out examples, because I think it has complex constructs that are not related to the functions that the library represents. And probably I have to add comments. 🤔

### CODE REVIEW

**Comment by evgenykochetkov** [comment]
`example-uid-equals` looks more like a functional test. I don't think the end user will find it useful.
The rest of examples are cool, but I think they will really benefit from brief comments explaining what is going on.

**Comment by brusherru** [comment]
@evgenykochetkov check out updated examples, please!

File: workspace/__lib__/xod-dev/pn532-nfc/example-read-page/patch.xodp [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+        "x": 340,
+        "y": 408
+      },
+      "size": {
+        "height": 51,
+        "width": 306
+      }
+    },
+    {
+      "content": "<- After initialization it will pulse continuously to trigger detecting of an NFC tag on each transaction",
```
Comment (evgenykochetkov) [inline workspace/__lib__/xod-dev/pn532-nfc/example-read-page/patch.xodp]: IMHO, this is way too technical. Do all users have to know about execution model and what a transaction is?

File: workspace/__lib__/xod-dev/pn532-nfc/example-nfc-scanner/patch.xodp [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+        "x": 340,
+        "y": 306
+      },
+      "size": {
+        "height": 51,
+        "width": 408
+      }
+    },
+    {
+      "content": "<- Pass pulse to update the led only if it our NFC tag",
```
Comment (evgenykochetkov) [inline workspace/__lib__/xod-dev/pn532-nfc/example-nfc-scanner/patch.xodp]: > it **is** our NFC tag

?

File: workspace/__lib__/xod-dev/pn532-nfc/example-nfc-scanner/patch.xodp [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+        "x": 340,
+        "y": 204
+      },
+      "size": {
+        "height": 51,
+        "width": 408
+      }
+    },
+    {
+      "content": "<- Check out the UID of your tag and then just bind\nthis values into `my-card` node",
```
Comment (evgenykochetkov) [inline workspace/__lib__/xod-dev/pn532-nfc/example-nfc-scanner/patch.xodp]: This looks like the last step, but user actually should start with it 🤔

File: workspace/__lib__/xod-dev/pn532-nfc/example-write-page/patch.xodp [hunk 4]
```diff
@@ -0,0 +1,350 @@
+{
+  "comments": [
+    {
+      "content": "<- Blink several times to notify that writing complete",
```
Comment (evgenykochetkov) [inline workspace/__lib__/xod-dev/pn532-nfc/example-write-page/patch.xodp]: Why not specify that it blinks 3 times?

File: workspace/__lib__/xod-dev/pn532-nfc/example-write-page/patch.xodp [hunk 5]
```diff
@@ -0,0 +1,350 @@
...
+        "x": 340,
+        "y": 102
+      },
+      "size": {
+        "height": 51,
+        "width": 306
+      }
+    },
+    {
+      "content": "<- After initialization it will pulse continuously to trigger detecting of an NFC tag on each transaction",
```
Comment (evgenykochetkov) [inline workspace/__lib__/xod-dev/pn532-nfc/example-write-page/patch.xodp]: Same as `example-write-page`

File: workspace/__lib__/xod-dev/pn532-nfc/example-write-page/patch.xodp (hunk 4 above)
Comment (evgenykochetkov) [reply workspace/__lib__/xod-dev/pn532-nfc/example-write-page/patch.xodp]: ... that writing **is** complete?

File: workspace/__lib__/xod-dev/pn532-nfc/example-write-page/patch.xodp [hunk 6]
```diff
@@ -0,0 +1,350 @@
...
+        "x": 340,
+        "y": 714
+      },
+      "size": {
+        "height": 51,
+        "width": 306
+      }
+    },
+    {
+      "content": "<- Pulse once when NFC tag detected",
```
Comment (evgenykochetkov) [inline workspace/__lib__/xod-dev/pn532-nfc/example-write-page/patch.xodp]: ... when NFC tag **is** detected?

**Pull request merged**